improver/regrid/landsea2.py

```diff
@@ -114,6 +117,12 @@ def process(self, cube_in: Cube, cube_in_mask: Cube, cube_out_mask: Cube) -> Cub
         Returns:
```

Comment:
Please update the doc-string to describe the expected behaviour when the output domain is not a subset of the input domain. Please also correct the error in the existing doc-string (`cube_our` does not exist!)

Reply:
done. thanks